Formula/fuego.rb

```diff
@@ -29,4 +29,8 @@ def install
                           "--with-boost=#{Formula["boost"].opt_prefix}"
     system "make", "install"
   end
+
+  test do
+    assert_match "maxgames", shell_output("#{bin}/fuego --help")
```

Comment:
If we pipe `genmove white\ngenmove black` to `fuego`, we can check that the output contains `Forced opening move`